text-to-sql-Shuchi/app.py

```
@@ -0,0 +1,121 @@
+    return output
+
+# Initialize the CodeAgent with the SQL engine tool
+model_name = 'Gemini Flash 2.0'
+if model_name.startswith("Gemini"):
+    model = LiteLLMModel(
+        model_id="gemini/gemini-2.0-flash-exp",
+        api_key=GEMINI_API_KEY
+    )
+    print("Using Gemini 2.0 Flash")
+else:
+    model = HfApiModel("Qwen/Qwen2.5-Coder-32B-Instruct")
```

Ruff (0.8.2)

Undefined name `HfApiModel`

(F821)